api/src/main/java/io/strimzi/api/kafka/model/KafkaConnector.java

```diff
@@ -51,7 +51,27 @@
                                 specReplicasPath = KafkaConnector.SPEC_REPLICAS_PATH,
                                 statusReplicasPath = KafkaConnector.STATUS_REPLICAS_PATH
                         )
-                )
+                ),
+                additionalPrinterColumns = {
+                        @Crd.Spec.AdditionalPrinterColumn(
+                                name = "Cluster",
+                                description = "The name of the Kafka Connect cluster this connector belongs to",
+                                jsonPath = ".metadata.labels.strimzi\\.io/cluster",
```

Comment:
```suggestion
                                jsonPath = ".metadata.labels." + Labels.STRIMZI_CLUSTER_LABEL,
```
Would changing it to this work still? :)

Reply:
No, because the escaping would not match.

Reply:
And because it's an annotation the value must be a compile time constant, so you can't do using a method to provide the quoting.

Reply:
Fair enough, I suspected as much, but thanks for taking the time to answer.